src/Illuminate/Queue/Worker.php

```diff
@@ -251,6 +254,29 @@ protected function handleJobException($connectionName, $job, WorkerOptions $opti
     }
 
     /**
+     * Mark the given job as failed if it has exceeded the maximum allowed attempts. This will likely be because
```

Comment:
Like this

```
    /**
     * Mark the given job as failed if it has exceeded the maximum allowed attempts. 
     * 
     * 
     * This will likely be because the job previously exceeded a timeout.
     *
     * @param  string  $connectionName
     * @param  \Illuminate\Contracts\Queue\Job  $job
     * @param  int  $maxTries
     * @return void
     */
```